tests/sentry/api/test_issue_search.py

```diff
@@ -129,30 +131,47 @@ def test_user(self):
             'me',
             [self.project],
             self.user,
-        ) == convert_user_value('me', [self.project], self.user)
+            None
+        ) == convert_user_value('me', [self.project], self.user, None)
 
     def test_team(self):
         assert convert_actor_value(
             '#%s' % self.team.slug,
             [self.project],
             self.user,
+            None,
         ) == self.team
 
     def test_invalid_team(self):
         assert convert_actor_value(
             '#never_upgrade',
             [self.project],
             self.user,
+            None,
         ).id == 0
 
 
 class ConvertUserValueTest(TestCase):
     def test_me(self):
-        assert convert_user_value('me', [self.project], self.user) == self.user
+        assert convert_user_value('me', [self.project], self.user, None) == self.user
 
     def test_specified_user(self):
         user = self.create_user()
-        assert convert_user_value(user.username, [self.project], self.user) == user
+        assert convert_user_value(user.username, [self.project], self.user, None) == user
 
     def test_invalid_user(self):
-        assert convert_user_value('fake-user', [], None).id == 0
+        assert convert_user_value('fake-user', [], None, None).id == 0
+
+
+class ConvertReleaseValueTest(TestCase):
+    def test(self):
+        assert convert_release_value('123', [self.project], self.user, None) == '123'
+
+    def test_latest(self):
+        release = self.create_release(self.project)
+        assert convert_release_value(
+            'latest',
```

Comment:
Do we let people make releases named latest?

Reply:
🤣 There are currently 146 releases with this name. Super great idea to call a release `latest`. This is just existing functionality, but we should probably guard against it... I'll make a ticket for it at least

Reply:
Perhaps `latest` should only be a meta value when there is no release named 'latest'?

Reply:
Won't tackle this for now, but it's documented here: https://getsentry.atlassian.net/browse/APP-1124